test/test_helpers/fake_hits.rb

```diff
@@ -1,8 +1,10 @@
+# frozen_string_literal: true
+
 module TestHelpers
   module FakeHits
     # Value is either number of hits or one of
     #
-    # :clickfest - all the hours, every day, get as many hits as is the current date (1..31)
+    # :clickfest - all the hours, every day, get a random number of hits (from 0 to 100)
```

Comment:
This made me lose my mind a bit, as I was looking at this report:
![daily_report](https://github.com/3scale/porta/assets/1270328/43f600f5-594c-43b7-928d-c4130e1ba4c2)

It took me quite some time to understand why the metrics totals show some positive numbers, but the Traffic graph shows all "zeroes"... Until I realized they were not zeroes, but rather they were all 14.

Now I made it random so the report shows nicely:
![image](https://github.com/3scale/porta/assets/1270328/0c25cc66-5688-4cae-9bf0-c212b83bfefd)

We do not check this in the test anyway, so I guess it's fine.